src/ui/RecipeForm.tsx

```diff
@@ -0,0 +1,112 @@
+import { FC, useState } from 'react';
+import { Formik, Form, Field, FieldArray } from 'formik';
+import styled from '@emotion/styled';
+import Input from './Input';
+import Select from './Select';
+import Ingredient from './Ingredient';
+import IngredientForm from './IngredientForm';
+import InputHHMM from './InputHHMM';
+import DirectionsBox from './DirectionsBox';
+import TextArea from './TextArea';
+
+const RecipeForm: FC = ({ className }) => {
+  //TODO: convert strings in ary to objs for label/id purposes
+  const bydelAry = [
```

Comment:
put this in a separate file, called like /src/util/constants.ts. Also, don't use data type in variable names. just follow naming conventions. for arrays i like to use the plural form of the data stored in the array. so here just `const bydels = [...];`

Reply:
actually i changed my mind, don't put it in a separate file, but just put constants at the top of the file underneath your imports. they don't need to be inside the function